packages/content/README.md

```diff
@@ -0,0 +1,148 @@
+# @esri/hub-content
+
+> Manage Hub content [`@esri/hub.js`](https://github.com/Esri/hub.js).
+
+### Example
+
+```bash
+npm install @esri/hub-content
+```
+
+```js
+import { getContent  } from '@esri/hub-content';
+
+// pass in a site id
+getContent("abc123")
+    .then(
+        contentModel => {
+            contentModel.title; // access content information
+        }
+    );
+```
+
+## Design
+
+Hub Content is the generic term for Hub-related items and integrated metadata. Content types include datasets, maps, documents, apps, surveys, as well as events, sites, pages, and initiatives. Content provides a common metadata model and interface for creating, viewing, and managing content - as well as content-specific attributes and associations such as map basemap, app datasets, site pages, and initiative site. 
+
+Content have common metadata, associated content or resources (e.g. owner member), and actions (update, delete). Content metadata is composed of the Portal Item + Data with content- or org-specific enrichments. For example, a Map content item has 
+- _literal metadata_ like `title`
+- _inferred metadata_ such as `coverage` (global, regional, local) derived from the geographic extent, 
+- _linked metadata_ `license` created through `licenseInfo`
+- _associated resources_ such as datasets via `operationalLayers` or `owner` (`as IHubMember`) via the `item.owner`
+
+### Methods
+
+- `getContent(id, associatedOptions, hubRequestOptions)`
+- `createContent(id, attributes, hubRequestOptions)`
+- `updateContent(id, attributes, hubRequestOptions)`
+- `deleteContent(id, hubRequestOptions)`
+
+`associatedOptions` is proposed to support external references that may require additional XHR to fetch. Similar to JSON-API or GraphQL. 
+For example `associatedOptions: { itemInfo: true, itemData: true, connectedContent: 5 }`
+
+- Question: should download formats + links be optional associations or require XHR via the download components?
+
+### Metadata
+
+Based on the [Hub Content UI Inventory](https://app.lucidchart.com/invitations/accept/dab82a25-2ed9-4f63-8075-3db8873087e3)
+
+Attemping a simple _Hungarian Notation_ for attributes suffix: `*Date`, `*Id`, `*Number`. Plural (e.g. `tags`) is an array.
+
+```ts
+export interface IContentModel extends IModel {
```

Comment:
I don't think this should extend `IModel` - I don't think we should include the redundant `item`